src/main/webapp/resources/js/components/project/ProjectSelect.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { Select, SelectProps, Tag, Typography } from "antd";
+
+export type Project = { identifier: number; name: string };
+
+export interface ProjectSelectProps extends SelectProps {
+  projects: Project[];
+}
+
+/**
+ * React component for displaying a project drop-down menu.
+ * @param {list} projects - list of projects that is to be displayed
+ * @param {function} onChange - function that is called when select option has changed
+ * @param {number} defaultValue - project identifier of the project that is to be displayed by default
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export function ProjectSelect({
+  projects,
+  onChange = undefined,
```

Comment:
I doubt you want the `onChange` to ever be `undefined`

Reply:
Please see 8023bed5bf9acf057c2433e225379d0c850cc7fe.